presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraClientConfig.java

```diff
@@ -157,6 +158,18 @@ public CassandraClientConfig setNativeProtocolPort(int nativeProtocolPort)
         return this;
     }
 
+    public boolean getUseSSL()
+    {
+        return useSSL;
+    }
+
+    @Config(("cassandra.use-ssl"))
```

Comment:
suggest: cassandra.ssl.enabled (like in HiveClientConfig, hive.s3.ssl.enabled)
With corresponding methods/variable name changes (isSslEnabled, etc)